### PR TITLE
code block formatting

### PR DESCRIPTION
**Pull request by uvtc**
made them agree with recent assets formatting cleanups (mostly tabs --> 2 space indents). Also used some markdown in comments to clarify between code and prose.

### CODE REVIEW

File: content/05-expression.md [hunk 1]
```diff
@@ -207,10 +208,10 @@ Object declaration begins with an opening curly brace `{` after which `key:value
 
 ```haxe
 {
-	key1:value1,
-	key2:value2,
-	...
-	keyN:valueN
+  key1:value1,
```
Comment (Aurel300) [inline content/05-expression.md]: Let's have `key: value` pairs, with a space after the colon. The text immediately above this should also be changed.

Comment (uvtc) [reply content/05-expression.md]: Ah, good catch. Thanks.

**Comment by Aurel300** [comment]
Thank you!

**Comment by uvtc** [comment]
You're welcome!

**Pull request merged**